Converters/Civil3d/Speckle.Converters.Civil3dShared/ToSpeckle/Raw/AlignmentSubentityArcToSpeckleRawConverter.cs

```diff
@@ -0,0 +1,98 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Sdk;
+
+namespace Speckle.Converters.Civil3dShared.ToSpeckle.Raw;
+
+public class AlignmentSubentityArcToSpeckleRawConverter : ITypedConverter<CDB.AlignmentSubEntityArc, SOG.Arc>
+{
+  private readonly ITypedConverter<AG.Plane, SOG.Plane> _planeConverter;
+  private readonly IConverterSettingsStore<Civil3dConversionSettings> _settingsStore;
+
+  public AlignmentSubentityArcToSpeckleRawConverter(
+    ITypedConverter<AG.Plane, SOG.Plane> planeConverter,
+    IConverterSettingsStore<Civil3dConversionSettings> settingsStore
+  )
+  {
+    _planeConverter = planeConverter;
+    _settingsStore = settingsStore;
+  }
+
+  public SOG.Arc Convert(object target) => Convert((CDB.AlignmentSubEntityArc)target);
+
+  public SOG.Arc Convert(CDB.AlignmentSubEntityArc target)
+  {
+    string units = _settingsStore.Current.SpeckleUnits;
+
+    // calculate midpoint of chord as between start and end point
+    AG.Point2d chordMid =
+      new((target.StartPoint.X + target.EndPoint.X) / 2, (target.StartPoint.Y + target.EndPoint.Y) / 2);
+
+    // calculate sagitta as radius minus distance between arc center and chord midpoint
+    double sagitta = target.Radius - target.CenterPoint.GetDistanceTo(chordMid);
+
+    // get unit vector from arc center to chord mid
+    AG.Vector2d midVector = target.CenterPoint.GetVectorTo(chordMid);
+    AG.Vector2d unitMidVector = midVector.DivideBy(midVector.Length);
+
+    // get midpoint of arc by moving chord mid point the length of the sagitta along mid vector
+    // if greater than 180 >, move in other direction of distance radius + radius - sagitta
+    // in the case of an exactly perfect half circle arc...🤷‍♀️
+    AG.Point2d midPoint = chordMid.Add(unitMidVector.MultiplyBy(sagitta));
+    try
+    {
+      if (target.GreaterThan180) // this can throw : The property gets an invalid value according to the entity's constraint type.
+      {
+        midPoint = chordMid.Add(unitMidVector.Negate().MultiplyBy(2 * target.Radius - sagitta));
+      }
+    }
+    catch (Exception e) when (!e.IsFatal()) { } // continue with original midpoint if GreaterThan180 doesn't apply to this arc
```

Comment:
Do we know if this will throw something specific? We don't really want to catch all non-fatal exceptions if we can avoid it.